src/networks.ts

```diff
@@ -177,6 +178,16 @@ export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
     oftEid: OFT_NO_EID,
     hypDomainId: MAINNET_CHAIN_IDs.INK,
   },
+  [CHAIN_IDs.TATARA]: {
+    name: "Katana Tatara",
+    family: NONE, // Snowflake family - tbd
+    nativeToken: "ETH",
```

Comment:
Is the native token ETH?

Reply:
Yep - you can see ETH spent in the deployment txn here: https://explorer.tatara.katana.network/tx/0x3906800775bc9114d1854cb254314e2a41e16094c273b95cf31564582929a2f6